components/TranscriptionView.tsx

```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { View } from "react-native";
+import {View, Text} from "react-native";
 
-const TranscriptionView = (): JSX.Element => {
-	return <View></View>;
+const TranscriptionView = (transcribedText:string): JSX.Element => {
```

Comment:
The first parameter of a react function component is always an object, called `props` by convention. The object contains all the props. I'd recommend defining an interface for typescript to be happy:

```ts
interface TranscriptionViewProps {
	transcribedText: string;
}
```

And then you would change this line to

```ts
const TranscriptionView = (props:TranscriptionViewProps): JSX.Element => {
```

See the PhotoButton component for an example.

Reply:
Oh okay, so this way the prop can have multiple attributes?